docs/running.md

```diff
@@ -245,35 +245,49 @@ az functionapp config appsettings list -g {resource-group-name} -n {function-app
 Replace `{resource-group-name}` and `{function-app-name}` with their respective values in the
 command above.
 
-##### Running Administrative Functions Locally
+##### Triggering Dataflow Functions with HTTP Triggers Locally
```

Comment:
Is the intent to never have administrative functions in the API? To me this seems like a different section, or a nuance, not necessarily a replacement.